SR2020/2020-06-18-version-0-2-0-simulator.md

```diff
@@ -0,0 +1,23 @@
+---
+to: sr2020-teams
+subject: Virtual Competition Simulation Version 2 Released
+---
+
+Hi
+
+We have released version 0.2.0 of the competition simulation which you can download from the [Docs page](https://studentrobotics.org/docs/competition-simulator/).
```

Comment:
Are the forms not a better place for this announcement?

Reply:
It'll be announced there too. The issue is that the forums aren't read as regularly by some teams, so an email deals with both ensuring everyone gets the message, and prompting those who haven't signed up yet.

Reply:
Sorry, I totally missed that this was an email going out to teams, not volunteers.  Ignore me.